app/uk/gov/hmrc/customs/api/common/connectors/ServiceLocatorConnector.scala

```diff
@@ -25,7 +25,7 @@ import uk.gov.hmrc.customs.api.common.domain.Registration
 import uk.gov.hmrc.http._
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
 import scala.concurrent.Future
```

Comment:
I think this should be something like `import scala.concurrent.{ExecutionContext, Future}`, not `import play.api.libs.concurrent.Execution.Implicits.defaultContext`